src/main/java/com/investmenttracker/enterprise/dao/InvestmentDAOStub.java

```diff
@@ -11,30 +11,30 @@
 @Repository
 public class InvestmentDAOStub implements IInvestmentDAO {
 
-    Map<Integer, investment> allInvestments = new HashMap<>();
+    Map<Integer, Investment> allInvestments = new HashMap<>();
 
     @Override
-    public investment save(investment investment) {
+    public Investment save(Investment investment) {
         return null;
     }
 
     @Override
     public Object fetch(int id) {
-        return null;
+        return allInvestments.get(id);
```

Comment:
I made this return what I think you wanted it to return. If you did want it to return null it is encouraged to add the @nullable annotation to fields or methods that return null per best practices. https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3